zephyr/lib/pm_runtime.c

```diff
@@ -90,8 +92,10 @@ void platform_pm_runtime_disable(uint32_t context, uint32_t index)
 		tr_dbg(&power_tr, "putting prevent on d0i3");
 		pm_policy_state_lock_get(PM_STATE_RUNTIME_IDLE, PM_ALL_SUBSTATES);
 		/* Disable power gating when preventing */
+#ifdef CONFIG_ACE
 		DSPCS.bootctl[PLATFORM_PRIMARY_CORE_ID].bctl |=
 			DSPBR_BCTL_WAITIPCG | DSPBR_BCTL_WAITIPPG;
+#endif
```

Comment:
I will verify it, but I think it's safe to delete this. This was already discussed when we have rename these registers, but the topic died later.

Reply:
removed now